src/udev/udevadm-control.c

```diff
@@ -77,7 +77,7 @@ int control_main(int argc, char *argv[], void *userdata) {
         }
 
         if (argc <= 1)
-                log_error("Option missing");
+                return help();
```

Comment:
No, please don't do this. Printing the full help is just not helpful, if we know what exactly is wrong. We don't do this anywhere else in the code base.

Reply:
OK. I will drop that part.
> We don't do this anywhere else in the code base.

cryptsetup and veritysetup.c do that. But, yeah, normally they are not run by user.